addon/templates/components/bs-navbar.hbs

```diff
@@ -0,0 +1,12 @@
+<div class="container-fluid">
```

Comment:
Hm, this does not have to be fluid, you could also want to have a `.container` container! Maybe introduce a `fluid` boolean property?